package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "libgit2",
+  "version": "0.27.0",
+  "repo": "https://github.com/libgit2/libgit2",
+  "desciption": " A cross-platform, linkable library implementation of Git that you can use in your application.",
```

Comment:
I think this is a typo and should be `description`?

Reply:
Yup, my mistake -- just fixed this.